src/gui/mzroll/forms/peakdetectiondialog.ui

```diff
@@ -202,31 +216,17 @@
             </property>
            </widget>
           </item>
-          <item row="1" column="3">
-           <widget class="QDoubleSpinBox" name="rtMin">
-            <property name="maximum">
-             <double>999999999.000000000000000</double>
-            </property>
-           </widget>
-          </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Mass Domain Resolution</string>
             </property>
            </widget>
           </item>
-          <item row="0" column="3">
-           <widget class="QDoubleSpinBox" name="mzMin">
-            <property name="maximum">
-             <double>999999999.000000000000000</double>
-            </property>
-           </widget>
-          </item>
-          <item row="1" column="2">
-           <widget class="QLabel" name="label_18">
+          <item row="2" column="0" colspan="2">
+           <widget class="QCheckBox" name="mustHaveMs2">
```

Comment:
@saifulbkhan If I check the box and reopen El-MAVEN, the box goes back to its unchecked state. Is that by design?

Reply:
Okay, I see the advantage. If I load LCMS data in the next session, I would not want this box checked automatically. This is fine for now.

Reply:
I do not recall making this decision explicitly. For it to save its checked state we might have to introduce a variable for this in the settings file as well. I can do that, if that is what we want.

Reply:
@saifulbkhan Two ways to handle the situation I mentioned:
1. Disable/enable the checkbox based on LCMS/DDA data
2. Keep the behavior as it is but save the state in emDB

Reply:
Got it. Will change it accordingly.